linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/service/AbstractDbMetaService.java

```diff
@@ -67,6 +68,12 @@ public List<MetaColumnInfo> getColumns(
     return this.getConnAndRun(operator, params, conn -> this.queryColumns(conn, database, table));
   }
 
+  @Override
+  public GenerateSqlInfo getSparkSql(
```

Comment:
It is not recommended to add Spark-related methods to AbstractDbMetaService, and a new interface should be added

Reply:
Ok, thank you for reviewing the code